src/EncompassRest/ClientParameters.cs

```diff
@@ -70,6 +70,11 @@ public int TimeoutRetryCount
         /// </summary>
         public UndefinedCustomFieldHandling UndefinedCustomFieldHandling { get; set; }
 
+        /// <summary>
+        /// The URL to call for API calls.Defaults to "https://api.elliemae.com/".
```

Comment:
nit: could you put a space between the period and Defaults?

Reply:
Done.